vulcan/_vulcan.py

```diff
@@ -166,14 +152,23 @@ def get_attendance(self, date_from=None, date_to=None):
     def get_messages(self, date_from=None, date_to=None):
         """
         Fetches messages from the given date
-
         Args:
             date_from (:class:`datetime.date`): Date, from which to fetch messages, if not provided
                 it's using the semester (period) start date
             date_to (:class:`datetime.date`): Date, to which to fetch messages, if not provided
                 it's using the semester (period) end date
-
         Yields:
             :class:`vulcan._message.Message`
         """
         return Message.get(self._api, date_from, date_to)
+
+    def send_message(self, teachers_id, title, content):
```

Comment:
I think that `teachers` would be a better parameter name. And it should accept ids, as well as `Teacher` objects (and maybe teacher names too?)